src/index.html

```diff
@@ -11,7 +11,160 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
     <script type="text/javascript" src="./main.js"></script>
+    <form action="./main.js" method="post">
+
+      <fieldset class="form-fieldset">
```

Comment:
```suggestion
    <form action="./main.js" method="post">
      <fieldset class="form-fieldset">
```
Don't leave empty lines between parent and child elements